archive_updater.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#still working through a couple of things and haven't tested everything, but this is a rough draft of the changes to come
+
+import subprocess
+import os
+
+def update_archive(archive_path:str):
```

Comment:
nit: python style usually says to have a space between names and the annotations, e.g.
```suggestion
def update_archive(archive_path: str):
```

Ditto for the lines below.